doc/user-guide/src/SUMMARY.md

```diff
@@ -3,9 +3,9 @@
 [Introduction](index.md)
 - [Installation](installation/index.md)
     - [Windows](installation/windows.md)
-        - [MSVC Prerequistes](installation/windows-msvc.md)
+        - [MSVC prerequistes](installation/windows-msvc.md)
```

Comment:
Note, there's a pre-existing typo here, it should be "prerequisites". Can you fix that here?

Reply:
Done!